helix-core/src/main/java/org/apache/helix/manager/zk/ParticipantManager.java

```diff
@@ -178,89 +194,18 @@ private void createLiveInstance() {
       }
     }
 
-    boolean retry;
-    do {
-      retry = false;
-      try {
-        _zkclient.createEphemeral(liveInstancePath, liveInstance.getRecord());
-        LOG.info("LiveInstance created, path: " + liveInstancePath + ", sessionId: " + liveInstance.getEphemeralOwner());
-      } catch (ZkNodeExistsException e) {
```

Comment:
Although it should be impossible that we see this exception anymore, shall we still catch it and throw HelixException?

Reply:
Or we can keep the wait and retry logic.

Reply:
The wait and retry logic is kept.